target/imagebuilder/Makefile

```
@@ -43,24 +43,29 @@ $(BIN_DIR)/$(IB_NAME).tar.zst: clean
 
 	$(INSTALL_DIR) $(PKG_BUILD_DIR)/packages
 
-ifeq ($(CONFIG_IB_STANDALONE),)
 ifneq ($(CONFIG_USE_APK),)
+  ifeq ($(CONFIG_IB_STANDALONE),)
 	$(call FeedSourcesAppendAPK,$(PKG_BUILD_DIR)/repositories)
 	$(VERSION_SED_SCRIPT) $(PKG_BUILD_DIR)/repositories
+  endif
+
+	$(INSTALL_DATA) ./files/README.md.apk $(PKG_BUILD_DIR)/packages/README.md
 else
+  ifeq ($(CONFIG_IB_STANDALONE),)
 	echo '## Remote package repositories' >> $(PKG_BUILD_DIR)/repositories.conf
 	$(call FeedSourcesAppendOPKG,$(PKG_BUILD_DIR)/repositories.conf)
 	$(VERSION_SED_SCRIPT) $(PKG_BUILD_DIR)/repositories.conf
 
+  endif
+
 	# create an empty package index so `opkg` doesn't report an error
 	touch $(PKG_BUILD_DIR)/packages/Packages
-	$(INSTALL_DATA) ./files/README.md $(PKG_BUILD_DIR)/packages/
+	$(INSTALL_DATA) ./files/README.md.opkg $(PKG_BUILD_DIR)/packages/README.md
```

```suggestion
	$(INSTALL_DATA) ./files/README.opkg.md $(PKG_BUILD_DIR)/packages/README.md
```
BTW is there any reason to not keep the `.md` suffix, so the file type detection based on filename continue to work?